### PR TITLE
Open project from project name

### PR DESCRIPTION
**Pull request by chiaberry**
Frontend changes: https://deploy-preview-310--atd-moped-main.netlify.app/moped

Hides the project_id column and instead projects can be opened from their Project Name. 

_Before_
Screen Shot 2021-05-18 at 11 04 58 AM

✨ After
Screen Shot 2021-05-18 at 11 06 03 AM

### CODE REVIEW

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js [hunk 1]
```diff
@@ -52,15 +55,27 @@ export const ProjectsListViewQueryConf = {
     project_name: {
       searchable: true,
       sortable: false,
+      link: "project_id",
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js:58]: the name of the column that has the link url information, in this case the project_id pk

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js [hunk 2]
```diff
@@ -52,15 +55,27 @@ export const ProjectsListViewQueryConf = {
...
       },
-      width: "20%",
+      width: "*",
       type: "String",
+      filter: values => {
+        const jsonValues = JSON.parse(values)
+        return (
+          <RouterLink
+            to={`/${jsonValues.singleItem}/${jsonValues.link}`}
+            className={"MuiTypography-colorPrimary"}
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js:73]: overrides the visited link turning purple

File: moped-editor/src/components/GridTable/GridTable.js [hunk 3]
```diff
@@ -451,7 +465,12 @@ const GridTable = ({ title, query }) => {
                                           )}
                                         </RouterLink>
                                       )
-                                    ) : isAlphanumeric(column) ? (
+                                    ) : query.config.columns[column]?.link ? (
+                                      query.getFormattedValue(
+                                        column,
+                                        buildLinkData(row, column))
+                                    )
+                                    : isAlphanumeric(column) ? (
```
Comment (chiaberry) [inline moped-editor/src/components/GridTable/GridTable.js:473]: check to see if the column should be turned into a link, and format the information based on the query

**Review by mateoclarke — APPROVED** [review]
link_approves

**Review by sergiogcx — APPROVED** [review]
Thanks for working on this! 🙏

**Comment by amenity** [comment]
Niiiice. @johnclary, your dreamz are coming true !! 🌈

**Review by amenity — APPROVED** [review]

**Pull request merged**